### PR TITLE
add types for `fancade-editor`

### PR DESCRIPTION
**Pull request by bricked-dev**
Creates a new definition package for [Fancade's in-game scripting editor](https://www.fancade.com/wiki/fancade-web#editor-scripting).

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/fancade-editor-script/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "ES5"
```
Comment (jakebailey) [inline types/fancade-editor-script/tsconfig.json:5]: This should match the expected config; see the bot warning and the README.

**Review by jakebailey — COMMENTED** [review]
Why not just call it `fancade-editor`?

**Comment by brycked** [comment]
oh you're right! gonna update the pr tomorrow 👍

**Comment by brycked** [comment]
reamed the package, sorry for accidentally deleting it in that one commit

File: types/fancade-editor-script/tsconfig.json (hunk 1 above)
Comment (brycked) [reply types/fancade-editor-script/tsconfig.json]: I made the package `es6` again, so it should match the template now.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@brycked thanks!

**Comment by brycked** [comment]
thanks everyone for reviewing!

**Comment by brycked** [comment]
Ready to merge

**Pull request merged**